src/AppInstallerRepositoryCore/Microsoft/Schema/1_0/SearchResultsTable_1_0.cpp

```diff
@@ -157,12 +136,18 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_0
         From().BeginParenthetical();
 
         // Add the field specific portion
-        int bindIndex = BuildSearchStatement(builder, field, match);
+        std::optional<int> bindIndex = BuildSearchStatement(builder, field, match);
+
+        if (!bindIndex)
+        {
+            AICLI_LOG(Repo, Verbose, << "ApplicationMatchField not supported in this version: " << ApplicationMatchFieldToString(field));

```

Comment:
>"ApplicationMatchField not supported in this version: " [](start = 40, length = 55)

Do we want to surface this back to user in some form?

Reply:
No, this is how a 1.0 database will avoid attempting to search for the new fields.  It is an intended short circuit, but is logged for informational purposes.

---
In reply to: [488314712](https://github.com/microsoft/winget-cli/pull/564#discussion_r488314712) [](ancestors = 488314712)